### ISSUE TITLE
Adding Imagesharp-beta7 causes exception

### ISSUE THREAD

**Issue by matyasforian** [mellinoe/veldrid#240]
I need the latest Imagesharp because of a bugfix. When I upgrade it to beta 7 the following code throws an exception:
```
var texPath = Path.Combine(AppContext.BaseDirectory, "Assets", imageName);
var imTex = new ImageSharpTexture(texPath, false);
```
The exception is: 
```
System.MissingMethodException: Method not found: 'SixLabors.ImageSharp.Image`1<SixLabors.ImageSharp.PixelFormats.Rgba32> SixLabors.ImageSharp.Image.Load(System.String)'.
   at Veldrid.ImageSharp.ImageSharpTexture..ctor(String path, Boolean mipmap)
   at Engine.ImageManager.Load(String imageName) ...
```
The issue is not present with ImageSharp beta 6. I am using Veldrid 4.7.0, .NET Core 3

**Comment by matyasforian** [comment]
The workaround is not to use `new ImageSharpTexture(string, bool)`, but load the image manually and then use it:
```
Image<Rgba32> im = Image.Load<Rgba32>(texPath);
var imTex = new ImageSharpTexture(im, false);
```

**Comment by feliwir** [comment]
This was fixed by #246 and can be closed @mellinoe
